src/System/Logger/Types.hs

```diff
@@ -84,6 +84,7 @@ import Control.Monad.Trans.Control
 import Control.Monad.Trans.Either
 import Control.Monad.Trans.State
 import Control.Monad.Trans.Trace
+import Control.Monad.Unicode
```

Comment:
My editor, too, is trying to force me into this. So far I resisted, but having now two against me I give up :-)